code/game/turfs/walls/resin.dm

```diff
@@ -122,6 +123,8 @@
 	var/multiplier = 1
 	if(I.damtype == BURN) //Burn damage deals extra vs resin structures (mostly welders).
 		multiplier += 1
+	if(I.damtype == BRUTE) //Basically all melee attacks will do more damage.
+		multiplier += 0.5
```

Comment:
2 things: firstly this should be an else if
secondly the comment is wrong? remove it or reword it to explain this properly